scripts/rank_models.py

```
@@ -0,0 +1,470 @@
+# ---------------------------------------------------------------------------
+# Chain ranking
+# ---------------------------------------------------------------------------
+
+
+@dataclass
+class RankReason:
+    provider: str
+    model: str
+    score: float
+    reason: str
+    I: float = 0.0
```

Ruff (0.15.18)

[error] Ambiguous variable name: `I`

(E741)